pkg/api/controllers/profile.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 Huawei Technologies Co., Ltd. All Rights Reserved.
+// Copyright 2019 The OpenSDS Authors to "Copyright 2019 The OpenSDS Authors"
```

Comment:
Please update the copyright header.

Reply:
Done i changed from  Copyright (c) 2019 Huawei Technologies Co., Ltd. All Rights Reserved. to Copyright 2019 The OpenSDS Authors to "Copyright 2019 The OpenSDS Authors"